recipes/alchemiscale/meta.yaml

```diff
@@ -0,0 +1,102 @@
+{% set name = "alchemiscale" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: alchemiscale
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/alchemiscale/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d46f372adcaa257eb909587f668e791915188795b60db0c11bbfd6bfe9ac1a84
+
+build:
+  number: 0
+  skip: true
```

Comment:
```suggestion
```

This is why!

Reply:
Lol, yes that was a mistake and explains why it was working for me locally before I made the PR